solidity_deposit_contract/README.md

```diff
@@ -0,0 +1,25 @@
+# eth2-deposit-contract
+
+## History
+
+This is a rewrite of the [Vyper Eth 2.0 deposit contract](https://github.com/ethereum/eth2.0-specs/blob/v0.12.2/deposit_contract/contracts/validator_registration.vy) to Solidity.
+
+The original motivation was to run the SMTChecker and the new Yul IR generator option (`--ir`) in the compiler.
+
+As of June 2020, this contract (the version tagged as [`r1`](https://github.com/axic/eth2-deposit-contract/tree/r1) in GitHub repository [axic/eth2-deposit-contract](https://github.com/axic/eth2-deposit-contract/tree/r1)) has been verified and is considered for adoption.
```

Comment:
Linking to the external axic repo might be confusing to leave in the README. I would consider a rewrite to be spec-repo centric or at least mention the migration and deprecation of the original repo

Reply:
I just rewrote it a bit. post-merge revise is welcome!